firestore.indexes.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Can you clarify whether this is actually meant to be the eventual Firestore Index configuration, or if it's just a placeholder? If it matches what we plan on deploying, I'm not clear on why there are `name` and `owner_name` fields.

We are not collecting any user information, like names.

Reply:
We have added this for testing purpose and added in .gitignore.

Reply:
Since this file is in gitignore, please remove it from this PR